### ISSUE TITLE
IMXB BHs accrete tiny mass

### ISSUE THREAD

**Issue by akonstantinu** [TeamCOMPAS/COMPAS#889]
**Describe the bug**
I created a plot of IMXB BHs accreted mass, and it turned out that it is very small.  We might incorrectly using the thermal timescale to determine the rate of mass loss from a MS donor. 

**Label the issue**
Please label the 'severity' and 'urgency' of this issue. You can choose:

`urgency_moderate` - This is a moderately urgent issue

`severity_minor`    - This is a minor bug with minimal impact

**To Reproduce**
`COMPAS --random-seed 1 --number-of-systems 10000 --metallicity 0.0142 --kick-magnitude-distribution MULLERMANDEL --remnant-mass-prescription  MULLERMANDEL --muller-mandel-kick-multiplier-NS 520 --logfile-definitions COMPAS_Output_Definitions.txt`

Together with the following COMPAS_Output_Definitions.txt
```

BSE_RLOF_Rec += { BINARY_PROPERTY::SYSTEMIC_SPEED, 
    STAR_1_PROPERTY::LUMINOSITY,
    STAR_2_PROPERTY::LUMINOSITY,
    STAR_1_PROPERTY::TEMPERATURE,
    STAR_2_PROPERTY::TEMPERATURE,
    STAR_1_PROPERTY::ANGULAR_MOMENTUM,
    STAR_2_PROPERTY::ANGULAR_MOMENTUM,
    STAR_1_PROPERTY::DYNAMICAL_TIMESCALE,
    STAR_2_PROPERTY::DYNAMICAL_TIMESCALE,
    STAR_1_PROPERTY::THERMAL_TIMESCALE,
    STAR_2_PROPERTY::THERMAL_TIMESCALE,
    STAR_1_PROPERTY::NUCLEAR_TIMESCALE,
    STAR_2_PROPERTY::NUCLEAR_TIMESCALE,
    STAR_1_PROPERTY::MDOT,
    STAR_2_PROPERTY::MDOT,
    STAR_1_PROPERTY::MASS_TRANSFER_DIFF,
    STAR_2_PROPERTY::MASS_TRANSFER_DIFF}
```

Choose accreding IMXB BHs
```
IMXBRestrictions= ((((RLOFdata['Stellar_Type(1)<MT']==13)&(RLOFdata['Stellar_Type(1)>MT']==13))
        |((RLOFdata['Stellar_Type(1)<MT']==14)&(RLOFdata['Stellar_Type(1)>MT']==14))|
         ((RLOFdata['Stellar_Type(1)<MT']==13)&(RLOFdata['Stellar_Type(1)>MT']==14)))#The accretor is a BH or a NS
        &(RLOFdata['Mass(2)>MT']>2.5)#The donor is less than 2.5Mo after the MT
        &(RLOFdata['Mass(2)>MT']<8)
        &((RLOFdata['Mass(2)>MT']-RLOFdata['Mass(2)<MT'])<0) # the secondary losses mass
        &((RLOFdata['Stellar_Type(2)<MT']==0)|(RLOFdata['Stellar_Type(2)<MT']==1))#The donor is a MS star
        &((RLOFdata['RLOF(2)>MT']==1)|(RLOFdata['RLOF(2)<MT']==1))#The secondary overfeals the RL

        )

```
```
Dum=RLOFdata[IMXBRestrictions&(RLOFdata['Stellar_Type(1)>MT']==14)][['SEED','Mass(2)<MT']].drop_duplicates(subset=['SEED'],keep="last"
)['Mass(2)<MT']-RLOFdata[IMXBRestrictions&(RLOFdata['Stellar_Type(1)>MT']==14)][['SEED','Mass(2)>MT']].drop_duplicates(subset=['SEED'],keep="first")['Mass(2)>MT']
plt.hist(-Dum)
plt.xlabel('Mass Loss in IMXBs', fontsize='24')
```

**Expected behavior**
A larger mass accretion was expected

**Screenshots**
![image](https://user-images.githubusercontent.com/83845073/215057531-52f246e9-899c-4a1a-892d-c106771ea1f9.png)

**Versioning (please complete the following information):**
 COMPAS v02.34.04

**Comment by akonstantinu** [comment]
@ilyamandel

**Comment by ilyamandel** [comment]
Just to make sure -- are you taking the difference between BH mass before the last MT step and after the first MT step?  Why?  I would think you want to take the difference between the BH mass after the last MT step and before the first MT step...

**Comment by akonstantinu** [comment]
Yes you are right. It was by accident, but there is no significant change even if you do it correctly.

**Comment by akonstantinu** [comment]
@ilyamandel Now I am thinking about it, it might be better to wait until I fix the mass loss problem. It might be that the accretion mass is small because there are mass winds.

**Comment by ilyamandel** [comment]
Should be fixed by #1123 , but leaving open for now as a reminder to test explicitly.

**Comment by ilyamandel** [comment]
Tested; some compact objects accrete more than 1 solar mass during the LMXB phase, so finally closing this issue! :)
